### PR TITLE
Updated launch dedicated to work with AppImages

### PR DESCRIPTION
**Pull request by chrisBirmingham**
This is an update the the launch-dedicated.sh shell script to work with AppImages. It chooses the AppImage to execute based on the provided Mod value and provides the --server flag to the AppImage on execution.

### CODE REVIEW

**Comment by pchote** [comment]
The current script is there for people who compile and run the server from source. Replacing this with a version that requires the AppImages to be downloaded externally isn't really appropriate.

Can you please make this a new `launch-dedicated-appimage.sh` script instead?

**Comment by chrisBirmingham** [comment]
Thanks for the comment, I have reverted my changes on the original script and moved them into a new launch-dedicated-appimage.sh script.

File: launch-dedicated-appimage.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    printf "Invalid Mod value %s provided. Allowed values are ra, cnc and d2k" $Mod
+    exit 1
+fi
+
+if [ ! -f "$App" ]; then
+    printf "Could not find AppImage %s for Mod %s in the current folder. Has it been downloaded?" $App $Mod
+    exit 1
+fi
+
+eval "./$App" --server \
```
Comment (abcdefg30) [inline launch-dedicated-appimage.sh]: Our other scripts run this command in a loop, is there a reason why this does not (or does `eval` do something I'm not aware of)?

File: launch-dedicated-appimage.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+EnableSyncReports="${EnableSyncReports:-"False"}"
+
+if [ "$Mod" = "d2k" ]; then
+    App="OpenRA-Dune-2000-x86_64.AppImage"
+elif [ "$Mod" = "cnc" ]; then
+    App="OpenRA-Tiberium-Dawn-x86_64.AppImage"
+elif [ "$Mod" = "ra" ]; then
+    App="OpenRA-Red-Alert-x86_64.AppImage"
+else
+    printf "Invalid Mod value %s provided. Allowed values are ra, cnc and d2k" $Mod
```
Comment (abcdefg30) [inline launch-dedicated-appimage.sh]: I suppose we want to end the sentence with a fullstop.

**Review by abcdefg30 — COMMENTED** [review]
Sorry for the late review. Although untested the script looks good to me.
Can you squash the commits please?

File: launch-dedicated-appimage.sh (hunk 2 above)
Comment (chrisBirmingham) [reply launch-dedicated-appimage.sh]: Updated to include a fullstop

File: launch-dedicated-appimage.sh (hunk 1 above)
Comment (chrisBirmingham) [reply launch-dedicated-appimage.sh]: Hi. When I was testing my script I found the appimage would stop the server after a game had ended and then restart the server a few seconds later. I don't know if that functionality is the same for running the server directly or it is a specific feature of the app images.

**Comment by Mailaender** [comment]
Needs a rebase.

**Comment by Mailaender** [comment]
Actually it doesn't.

**Comment by Mailaender** [comment]
Can we enable `Squash and Merge` for the repository. That would solve this case.

**Comment by pchote** [comment]
In any case this would still need to be rebased and updated for the new options in launch-dedicated.sh.

**Comment by pchote** [comment]
This has been stalled for a long time, so closing to help clean up the PR queue.

**Pull request closed without merging**